test-workspace/tsc/vue3_strictTemplate/dataAttributes/main.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <div>
+        <svg><path stroke="#efefef" stroke-width="123" /></svg>
+        <foo stroke-width="123" />
+    </div>
+</template>
+
+<script setup lang="ts">
+import { defineComponent } from 'vue';
+
+const Foo = defineComponent({
+    props: {
+        strokeWidth: { type: String, required: true },
+    }
+});
+</script>
```

Comment:
Nit: could you please use tabs to indent to match this repo's code style? Thanks

```suggestion
<template>
	<div>
		<svg><path stroke="#efefef" stroke-width="123" /></svg>
		<foo stroke-width="123" />
	</div>
</template>

<script setup lang="ts">
import { defineComponent } from 'vue';

const Foo = defineComponent({
	props: {
		strokeWidth: { type: String, required: true },
	}
});
</script>
```